packages/@okta/vuepress-site/docs/guides/configure-ciba/main/index.md

```diff
@@ -0,0 +1,322 @@
+
+### Base64-encode the client ID and secret
+
+The call to the `/bc/authorize` endpoint and to the `/token` endpoint requires authentication. In this case, it’s a Basic Auth digest of the client ID and secret of the OIDC client that you created in [an earlier section of the guide](#create-an-oidc-client-integration-and-enable-CIBA). You can find them on the client’s **General** tab in the **Client Credentials** section.
+
+To use a Basic Authorization header in both the `/bc/authorize` and `/token` requests, Base64-encode the string and set it in the Authorization header:
+
+```sh
+   Authorization: Basic ${Base64(${clientId}:${clientSecret})}
+```
+
+### Initiate the flow
+
+The CIBA challenge request is sent by an OIDC client to an Okta authorization server. This request can include the following parameters:
+
+**Supported parameters**
+
+| Parameter                    | Value                                                                        |
+| :--------------------------- | :--------------------------------------------------------------------------- |
+| `binding_message`| (Optional) A human-readable message that appears on the authentication device to identify the transaction. |
+| `id_token_hint` | An ID token that was issued during initial user authentication. The token is passed back as a hint to identify the user for whom additional authentication is being requested. You can specify either `login_hint` or `id_token_hint` in the authentication request, not both. This parameter isn’t used in this use case example. |
+| `login_hint` | Information that identifies the user for whom authentication is being requested. This is typically the user’s email address. You can specify either `login_hint` or `id_token_hint` in the authentication request, not both. |
+| `request`| If you are using `private_key_jwt` as the [token endpoint authentication method](/docs/reference/api/apps/#credentials), this is the JWT created by the client that enables you to pass requests as a single, self-contained parameter. See [Parameter details](/docs/reference/api/oidc/#client-initiated-backchannel-authentication-parameter-details). |
+| `request_expiry`| (Optional) A positive integer that allows the client to request how long (in seconds) the authentication request is valid. This value is returned in the response from the authorization server as the `expires_in` parameter. If your request doesn’t contain this parameter, the default time that the request is valid is 300 seconds. |
+| `scope` | The `openid` scope is required for authentication requests. You can also include other [scopes](/docs/reference/api/oidc/#access-token-scopes-and-claims).|
+
+> **Note:** For additional request examples using different parameters as well as a public key/private key [client authentication](/docs/reference/api/oidc/#client-authentication-methods) example, see [CIBA request examples](/docs/reference/api/oidc/#ciba-request-examples).
```

Comment:
I didn't see this reference, ciba request examples. Is that in another PR?

Reply:
Yes, this is in Mymoona's CIBA PR #3709